README.md

```
@@ -1,71 +1,32 @@
+- **Common Torizon**: built on top of BSPs from third parties.
 
-See [docs/setup-environment.md](./docs/setup-environment.md) for how the build-environment setup scripts work and how to add support for a new vendor or board.
+Documentation
+=============
+All build guides and project documentation are unified on the Torizon OS docs
+site:
 
-Development Process
-===================
-Torizon is maintained by the Toradex R&D team. Development happens in this repository, including issues, PRs, and discussions. This repository is then used by our internal CI/CD infrastructure.
+### 📖 https://torizon.github.io//
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)